app/controllers/experiments_controller.rb

```diff
@@ -369,11 +371,53 @@ def create
         replication_level: [:optional, :security_default, :integer, :positive],
         execution_time_constraint: [:optional, :security_default, :integer, :positive],
         parameter_constraints: [:optional, :security_json],
-        type: [:optional, :security_default]
+        type: [:optional, :security_default],
+        workers_scaling: [:optional] # TODO boolean validator
```

Comment:
W takim razie dwie rzeczy:
- params[:workers_scaling] - wystarczy, że jest jakakolwiek wartość i wtedy jest true? Ostatnio podobna sprawa była z onsite_monitoringiem, które zmieniałem na "on" - to trzeba też będzie jakoś ustandaryzować i udokumentować
- params[:woreks_scaling_params] - nieudokumentowane; jako, że pełną dokumentację create musi napisać ktoś, kto pisał tą metodę, na razie napiszcie w komentarzu nad tą metodą o wszystkich parametrach związanych ze skalowaniem